journals/apps/core/views.py

```diff
@@ -108,4 +109,9 @@ def required_auth(request):
     """
     # Quoted URL: http%3A//localhost%3A18130/basket/add/%3Fsku%3D8CF08E5
     forwarded_url = unquote(request.GET.get('forward'))
+
+    (_, forwarded_url_domain, _, _, _) = urlsplit(forwarded_url)
+    if forwarded_url_domain not in get_whitelist_domains(request):
```

Comment:
@HammadAhmadWaqas 
can we change this check like this to remove empty string in `permitted_domains` below?
```
if forwarded_url_domain and forwarded_url_domain not in get_whitelist_domains(request):
```

Reply:
yes sure!

Reply:
I you still think its better idea to check , obviously I can add this check and remove empty string from `permitted_domains`

Reply:
Yeah i think we should remove it. As you can see `get_whitelist_domains ` method's name also reflects that this method is going to return list of domain and empty value in the list does not make sense there.